src/domain/challenge/hub/HubDashboardNavigation/useHubDashboardNavigation.ts

```diff
@@ -24,23 +26,40 @@ export interface DashboardNavigationItem {
   id: string;
   nameId: string;
   displayName: string;
+  tagline: string;
+  vision?: string;
   journeyTypeName: JourneyTypeName;
-  visualUri: string | undefined;
+  visual?: {
```

Comment:
what I don't agree is these ones, we are using alternative text, and also passing a visual to the `ChallengeCard` and `OpportunityCard` `banner` prop. 
It was me with Aleks the one who made banner a complex type uri and alternativeText, but that was also useful because they come this one directly from the API...
I can agree to change it back to bannerUri and bannerAltText, but we are going to need to change it in many places